hexrd/ui/workflow_selection_dialog.py

```diff
@@ -0,0 +1,36 @@
+from PySide2.QtCore import QObject
+
+from hexrd.ui.hexrd_config import HexrdConfig
+from hexrd.ui.ui_loader import UiLoader
+from hexrd.ui import constants
+
+
+class WorkflowSelectionDialog(QObject):
```

Comment:
I think this only needs to inherit from `QObject`, by the way, if it emits signals, or if we need to install an event filter. Otherwise, it can be just a regular class.